tests/integration/deploy_jenkins.py

```diff
@@ -30,14 +40,22 @@ class JenkinsDeployment:
 
     @property
     def hostname(self) -> str:
-        """Calculate the hostname."""
+        """Calculate the hostname.
+
+        Returns:
+            The hostname.
+        """
```

Comment:
Out of curiosity, would this be a list of addresses in str format?

Reply:
I think it's just one. Isn't it?

Reply:
I was wondering why the enclosure in square brackets in the following code:
`f"[{self.public_address}]"

Reply:
I was wondering why the enclosure in square brackets in the following code:
`f"[{self.public_address}]"

Reply:
Hadn't noticed that, but it's seems like single item